src/test/groovy/unit/org/pih/warehouse/inventory/TransactionTypeSpec.groovy

```diff
@@ -0,0 +1,91 @@
+package org.pih.warehouse.inventory
+
+import spock.lang.Unroll
+
+import grails.testing.gorm.DomainUnitTest
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Test
```

Comment:
please remove these junit imports. Also please remove the @Test annotations from each of the tests. You don't need them in Spock

Reply:
done